scripts/prepare_for_nuget_pack.sh

```diff
@@ -44,34 +52,34 @@ cd $PRODUCTS_DIR
 # Copy each platform
 make_dir_if_necessary "nuget/Debug-macosx"
 echo "Copy Debug-macosx Framework into nuget folder"
-rsync -a Debug/FluentUI.framework/ nuget/Debug-macosx/FluentUI.framework/
+rsync_excluding_swift_source_info_files Debug/FluentUI.framework/ nuget/Debug-macosx/FluentUI.framework/
 
 make_dir_if_necessary "nuget/Ship-macosx"
 echo "Copy Ship-macosx Framework into nuget folder"
-rsync -a Release/FluentUI.framework/ nuget/Ship-macosx/FluentUI.framework/
+rsync_excluding_swift_source_info_files Release/FluentUI.framework/ nuget/Ship-macosx/FluentUI.framework/
 
 make_dir_if_necessary "nuget/Debug-iphoneos"
 echo "Copy Debug-iphoneos build output into nuget folder"
 rsync -a Debug-iphoneos/libFluentUI.a nuget/Debug-iphoneos/
```

Comment:
nit: since we are using these path like 3 times with appended component should we just create an short variable so we avoid any human errors like typos?

Reply:
I felt the same way and was thwarted by my questionable bash skills. I'll take another stab at simplifying this.

Reply:
For now I'm going to skip improving this to prioritize getting a build going with these changes, but I may come back here to try and clean things up.